Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1153,7 +1153,7 @@ def plot2D(self,data1,data2,tmpl,gm,vtk_backend_grid=None,vtk_backend_geo=None):
       else:
         lmx= levs[-1]
       mapper.SetScalarRange(lmn,lmx)
-      returned["vtk_backend_luts"]=[[lut,[lmn,lmx,False]],]
+      returned["vtk_backend_luts"]=[[lut,[lmn,lmx,True]],]
```

Comment:
These map entries really need some documentation. I have no idea what this change should do.